projects/components/src/datagrid/datagrid.component.html

```diff
@@ -111,22 +111,30 @@ <h3 *ngIf="this.header" class="vcd-header">{{ header }}</h3>
             </div>
         </clr-dg-cell>
 
-        <clr-dg-cell *ngFor="let column of columnsConfig">
-            <!-- Default renderer -->
-            <ng-container *ngIf="column.fieldName">{{ restItem | nestedProperty: column.fieldName }}</ng-container>
+        <div *ngFor="let column of columnsConfig">
+            <ng-template #cellTemplate>
```

Comment:
If we can have an enabled switch on the clipped text directive, we won't need to dynamically add the directive, removing the duplication of HTML?